### PR TITLE
Add Calendar Events Synchronization / Test fixes

### PR DESCRIPTION
**Pull request by julianc269**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/FinancialDatabaseParser.java [hunk 1]
```diff
@@ -55,7 +55,7 @@ public Command parseCommand(String userInput) throws ParseException {
 
         final String commandWord = matcher.group("commandWord");
         final String arguments = matcher.group("arguments");
-        switch (commandWord.toLowerCase()) {
+        switch (commandWord) {
```
Comment (xiaoyeong) [inline src/main/java/seedu/address/logic/parser/FinancialDatabaseParser.java]: why did you change this part? the pervious one seem fine to me

Comment (julianc269) [reply src/main/java/seedu/address/logic/parser/FinancialDatabaseParser.java]: The previous one allowed for mixed case command words, causing some of the system tests to fail.
